src/main/kotlin/no/nb/mlt/wls/order/service/SynqOrderService.kt

```diff
@@ -57,20 +57,38 @@ class SynqOrderService(
     }
 
     suspend fun updateOrder(payload: ApiUpdateOrderPayload): ResponseEntity<SynqError> {
-        val uri = URI.create("$baseUrl/orders/batch")
-
+        // TODO - Should maybe just get a regular SynqOrderPayload here instead of converting it?
+        // If anything the regular order service should do the conversion either at call site or somewhere else
         val orders = SynqOrder(listOf(payload.toOrder().toSynqPayload()))
```

Comment:
I agree, it makes sense to move this out of the SynQ handling

Reply:
True but we can look at it during workshop today